mvc/views/razor.rst

```diff
@@ -1,8 +1,283 @@
-.. include:: /../common/stub-topic.txt
+Razor Syntax In MVC 6
+=====================
+By `John Rowley`_
 
-|stub-icon| Razor Syntax
-========================
+---------------
+What is Razor?
+---------------
+Razor is the ASP.NET view engine used in MVC. It allows c# and
+html to be mixed together in `Razor Markup` providing a simple and elegant
+way to render html.
 
-.. include:: /../common/stub-notice.txt
+Razor is the only view engine that MVC ships, but it is possible to use a
+different view engine.
 
-.. _issue: https://github.com/aspnet/Docs/issues/130
+Implicit Expressions
+^^^^^^^^^^^^^^^^^^^^^
+Implicit Expressions provide a simple mechanism to render data. They are simply
+an ``@`` followed by a variable or method. Razor renders the value to at
+runtime. The following markup renders the variable `` GuestName`` inside an H2
+heading element.
+
+.. code-block:: c#
+
+  <h2>@GuestName</h2>
+
+Razor automatically escapes the ``@`` used in an email address. To escape ``@``
+use ``@@``
+
+*********
+Explicit
+*********
+Sometimes you need to render variables without any space in between it and
```

Comment:
Your example later in this article `@await ____` conflicts with this statement. In essence `@await Foo` is a special implicit expression Syntax. May want to mention it as an exception to the rule.